components/popover/index.js

```diff
@@ -90,8 +79,15 @@ class Popover extends Component {
 		const handler = isListening ? 'addEventListener' : 'removeEventListener';
 
 		window.cancelAnimationFrame( this.rafHandle );
-		window[ handler ]( 'resize', this.throttledSetOffset );
-		window[ handler ]( 'scroll', this.throttledSetOffset, true );
+		window[ handler ]( 'resize', this.throttledComputePopoverPosition );
+		window[ handler ]( 'scroll', this.throttledComputePopoverPosition, true );
+	}
+
+	throttledComputePopoverPosition( event ) {
+		if ( event.type === 'scroll' && this.contentNode.current.contains( event.target ) ) {
```

Comment:
It took me more than a minute to grasp why this condition is here. It seems obvious in retrospect, but a comment would have been nice.